docs/architecture/adr-071-bank-v2.md

```diff
@@ -0,0 +1,102 @@
+# ADR-71 Bank V2
+
+## Status
+
+DRAFT
+
+## Changelog
+
+* 2024-05-08: Initial Draft (@samricotta, @julienrbrt)
+
+## Abstract
+
+The primary objective of refactoring the bank module is to simplify and enhance the functionality of the Cosmos SDK. Over time the bank module has been burdened with numerous responsibilities including transaction handling, account restrictions, delegation counting, and the minting and burning of coins. 
+
+In addition to the above, the bank module is currently too rigid and handles too many tasks, so this proposal aims to streamline the module by focusing on core functions `Send`, `Mint`, and `Burn`.
```

Comment:
I feel like it is a mistake to remove balances from the purview of the bank module.

Reply:
This was discussed during the working group call. The bank module will still keep track of balances.